docs/guide/topics/reactions.rst

```diff
@@ -0,0 +1,133 @@
+.. currentmodule:: discord
+
+.. _guide_topic_reactions
+
+Reactions
+==========
+
+Reactions are a feature within Discord where you can add emojis next to messages.
+They have many uses, but a common one for bots is for polls or giving feedback to a command without sending a new message.
+Anyone can react to a message.
```

Comment:
```suggestion
Members with the :attr:`~Permissions.add_reactions` can add new reactions to messages. However, this permission is not necessary to add to an existing reaction.
```

The note on line 31 also gets this across, but I feel like it conflicts with this sentence as it is.

Reply:
i feel like a note would be better than adding that, perhaps just removing the "anyone can react to a message"?

Reply:
> perhaps just removing the "anyone can react to a message"?

Yeah that seems good.